plugins/inputs/nginx_vts/README.md

```diff
@@ -0,0 +1,122 @@
+# Telegraf Plugin: nginx_vts
+
+This plugin gathers Nginx status using external virtual host traffic status module -  https://github.com/vozlt/nginx-module-vts. This is an Nginx module that provides access to virtual host status information. It contains the current status such as servers, upstreams, caches. This is similar to the live activity monitoring of Nginx plus.
+For module configuration details please see its [documentation](https://github.com/vozlt/nginx-module-vts#synopsis).
+
+### Configuration:
+
+```
+# Read nginx status information using nginx-module-vts module
+[[inputs.nginx_vts]]
+  ## An array of Nginx status URIs to gather stats.
+  urls = ["http://localhost/status"]
+```
+
+### Measurements & Fields:
+
+- nginx_vts_connections
+  - active
+  - reading
+  - writing
+  - waiting
+  - accepted
+  - handled
+  - requests
+- nginx_vts_server, nginx_vts_filter
+  - requests
+  - request_time
+  - in_bytes
+  - out_bytes
+  - response_1xx_count
+  - response_2xx_count
+  - response_3xx_count
+  - response_4xx_count
+  - response_5xx_count
+  - cache_miss
+  - cache_bypass
+  - cache_expired
+  - cache_stale
+  - cache_updating
+  - cache_revalidated
+  - cache_hit
+  - cache_scarce
+- nginx_vts_upstream
+  - requests
+  - request_time
+  - response_time
+  - in_bytes
+  - out_bytes
+  - response_1xx_count
+  - response_2xx_count
+  - response_3xx_count
+  - response_4xx_count
+  - response_5xx_count
+  - weight
+  - maxFails
+  - failTimeout
+  - backup
+  - down
+- nginx_vts_cache
+  - max_bytes
+  - used_bytes
+  - in_bytes
+  - out_bytes
+  - miss
+  - bypass
+  - expired
+  - stale
+  - updating
+  - revalidated
+  - hit
+  - scarce
+
+
+### Tags:
+
+- nginx_vts_connections
+  - server
+  - port
+- nginx_vts_server
+  - server
+  - port
+  - zone
+- nginx_vts_filter
+  - server
+  - port
+  - filter_name
+  - filter_key
+- nginx_vts_upstream
+  - server
+  - port
+  - upstream
+  - upstream_address
+- nginx_vts_cache
+  - server
+  - port
+  - zone
+
+
+### Example Output:
+
+Using this configuration:
+```
+[[inputs.nginx_vts]]
+  ## An array of Nginx status URIs to gather stats.
+  urls = ["http://localhost/status"]
+```
+
+When run with:
+```
+./telegraf -config telegraf.conf -input-filter nginx_vts -test
+```
+
+It produces:
+```
+* Plugin: inputs.nginx_vts, Collection 1
```

Comment:
For example output, we are trying to strip out the command used, the plugin prefix (`Plugin: inputs.thing`) and the leading `> `. Can you remove that from this output please.